cypress-tests/cypress/support/pageBuilder/pbCreateCategoryAndBlocks.ts

```diff
@@ -1,10 +1,16 @@
 import { gqlClient } from "../utils";
-
+import { createHeadingBlock } from "./pbCreateCategoryAndBlocks/createHeadingBlock";
 interface CreateCategoryAndBlocksParams {
     blockCategory: Record<string, any>;
     blockNames: string[];
+    content?: Content;
 }
 
+type Content = {
```

Comment:
It's un unwritten rule, but we prefer `interface` over `type` when possible. 

In this case, it's possible, so basically let's just switch `type` with `interface`.